apps/back/src/DataFixtures/AppFixtures.php

```diff
@@ -3,23 +3,33 @@
 namespace App\DataFixtures;
 
 use App\Entity\Courses;
+use App\Entity\User;
 use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Persistence\ObjectManager;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
 
 class AppFixtures extends Fixture
```

Comment:
Is your project using php 8.2 ?
In this case, the class can be readonly
https://php.watch/versions/8.2/readonly-classes

Reply:
My bad, i have this error message : "readonly' class cannot extend non-readonly class 'Fixture'"
when I set this class to readonly.